### PR TITLE
Migrate ACTION_LAYER_MOMENTARYs to MO()

### PR DESCRIPTION
**Pull request by fauxpark**
## Description

...

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by drashna** [comment]
Honestly, I'm not really sure how I feel about this PR.  

While I think that we do need to get away from using `action_get_macros` and `fn_actions`, I'm not sure this is the correct way to do so.

It looks like none of these are default keymaps, and I'm hesitant about changed personal keymaps.

**Comment by fauxpark** [comment]
This all stemmed from my discovery in https://github.com/qmk/qmk_firmware/pull/4762#issuecomment-451061476 that the space allocated to media keys is pretty much exhausted. As I mentioned in the next comment there is no need for the `KC_FN*` keycodes because they duplicate the first 32 `F()` keycodes, but I realised that `fn_actions` could be done away with entirely in order to simplify the code a little more.

I don't like editing other people's keymaps either, but I really think we should just get it over and done with so we can claw back some extra bytes (I'm not sure exactly how much space the whole `fn_actions` system costs though). Where possible I've tried to preserve alignment of the surrounding text to make as little impact as possible.

Is it possible to have the compiler raise a warning if it detects the use of `fn_actions`?
And perhaps I should make my next PR migrating `KC_FN*` to `F(*)` to at least free up those spots in keycode.h.

**Comment by fauxpark** [comment]
I found that adding the `deprecated` attribute to `fn_actions[]`, `action_function()` and `action_get_macro()` and setting the CFLAG `-Wno-error=deprecated-declarations` will generate a compiler warning without stopping the build. @skullydazed @drashna ?

**Comment by skullydazed** [comment]
I have changed the base from `master` to `future_unformatted` as part of the [Aug 2019 Breaking Change](https://github.com/qmk/qmk_firmware/issues/6146).

File: docs/ChangeLog/20190830/PR5176.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+* Migrating `ACTION_LAYER_MOMENTARY()` entries in `fn_actions` to `MO()` keycodes
+    * `fn_actions` is deprecated, and its functionality has been superseded by direct keycodes and `process_record_user()`
+    * The end result of removing this obsolete feature should result in a decent reduction in firmware size and code complexity
+    * All keymaps affected are recommended to switch away from `fn_actions` in favour of the new custom keycode/macro features
```
Comment (skullydazed) [inline docs/ChangeLog/20190830/PR5176.md]:
```suggestion
    * All keymaps affected are recommended to switch away from `fn_actions` in favour of the [custom keycode](https://docs.qmk.fm/#/custom_quantum_functions) and [macro](https://docs.qmk.fm/#/feature_macros) features
```

**Pull request merged**